### PR TITLE
Add support for effective date on plone.app.transmogrifier.datesupdater blueprint

### PR DESCRIPTION
**Pull request by hvelarde**

### CODE REVIEW

**Comment by hvelarde** [comment]
@mjpieters @rpatterson @thet can anyone of you take a quick look on this?

File: src/plone/app/transmogrifier/datesupdater.py [hunk 1]
```diff
@@ -45,11 +48,15 @@ def __iter__(self):
                 continue  # object not found
 
             creationdate = item.get(self.creationkey, None)
-            if creationdate and getattr(ob, 'creation_date', False):
+            if creationdate and hasattr(ob, 'creation_date'):
```
Comment (hvelarde) [inline src/plone/app/transmogrifier/datesupdater.py]: @thet see http://stackoverflow.com/a/24971134/644075

**Comment by rodfersou** [comment]
@hvelarde I didn't notice anything obscure about your pull request, for me we can merge it :)

**Comment by thet** [comment]
The expiration date is likewise, would you mind to add it too? I can't look deeper into this PR, because I'm on vacation.

**Comment by hvelarde** [comment]
@thet done; I also fixed the tests

**Comment by hvelarde** [comment]
@thet ping

**Comment by thet** [comment]
@hvelarde done!

**Comment by hvelarde** [comment]
release?

**Comment by thet** [comment]
@agitator has release rights

**Pull request merged**